### DISCUSSION TITLE
Forthcoming changes in Sentinel-2 products

### DISCUSSION THREAD

**Discussion by MullerMitro** [davidfrantz/force#85] [General]
Greetings,

Recently, ESA has announced that  deployment of Global Reference Image for new data produced will be initiated “during coming months” (May not be systematically applied).  Some changes to Sentinel-2 product format will be made. See link for news item and sample products:
https://sentinels.copernicus.eu/web/sentinel/-/forthcoming-deployment-of-the-copernicus-sentinel-2-products-geometric-refinement 

Will changes in data format affect utilization of FORCE in the near future? 

Best regards,
Mitro Müller

**Comment by davidfrantz** [comment]
Hi Mitro,

I am not expecting any issues with this. ESA constantly progresses with the data format specification. With only a few exceptions (which occurred in the first couple of years), these changes do not affect the processing.

But in case you want to be sure, the link says something about sample data.

Cheers,
David
